www/front_src/src/route-components/administration/extensions/manager/index.js

```diff
@@ -330,13 +330,13 @@ class ExtensionsRoute extends Component {
     } else if (!installed && !not_installed && !updated) {
       callback(params, nothingShown);
     } else {
-      if (!updated) {
+      if (updated) {
```

Comment:
is it the expected behaviour ?

Reply:
I don't know why but the behaviour was inverse when i touch the file.
I saw with Tom for the mechanism .
It s work like that.
The switches works fine! If you want we can meet for a demo :D